lunes_cms/cms/utils.py

```diff
@@ -61,11 +61,20 @@ def document_to_string(doc):
     :rtype: str
     """
     alt_words = [str(elem) for elem in doc.alternatives.all()]
+    has_foto = "\U0001F4F7" if doc.document_image.all() else ""
```

Comment:
Cool idea with the camera icon. However I think I would double-check with the creator of the issue to make sure we don't do anything unwanted. Did you already ask about it? :blush:

Reply:
I asked for check in the issue page 😸

Reply:
@JoeyStk 
Changed as disscussed in the issue page 👍

Reply:
yes, thank you :)